### PR TITLE
Add security issue creation

### PR DESCRIPTION
**Pull request by andreeleuterio**
[As discussed in Slack](https://sourcegraph.slack.com/archives/C032Z79NZQC/p1662043458595759) Security needs a release-blocker issue for every release so that we ensure CVEs are patched and documented before the release.

### CODE REVIEW

File: content/departments/engineering/dev/process/releases/release_issue_template.md [hunk 1]
```diff
@@ -31,6 +31,7 @@ Perform these steps three days before the release date to generate a stable rele
   ```sh
   yarn release release:status
   ```
+- [ ] Create a `Security release approval issue` and post a message in the [#security](https://sourcegraph.slack.com/archives/C1JH2BEHZ) channel tagging @security-support.
```
Comment (coury-clark) [inline content/departments/engineering/dev/process/releases/release_issue_template.md:34]: Is this different than the 5 day prior security image scan that security is doing?

Comment (andreeleuterio) [reply content/departments/engineering/dev/process/releases/release_issue_template.md:34]: The scans happen nightly. We need to cross-check the last scan and create a table with accepted CVEs and justifications such as https://handbook.sourcegraph.com/departments/security/tooling/trivy/3-43-1/.

It's a need for strategic customers that we provide them these justifications along with the release. We will look into automating portions of this from the our side.

Comment (unknwon) [reply content/departments/engineering/dev/process/releases/release_issue_template.md:34]: We already have a 5-day prior event for minor releases, and based on the discussion on Slack, I think this check item should be actually added to the patch release template instead? https://github.com/sourcegraph/handbook/blob/andre/release-process-update/content/departments/engineering/dev/process/releases/patch_release_issue_template.md

Comment (andreeleuterio) [reply content/departments/engineering/dev/process/releases/release_issue_template.md:34]: I missed the patch release template, we want this in place for both. The event is helpful but doesn't serve as a gate for the release.

Comment (andreeleuterio) [reply content/departments/engineering/dev/process/releases/release_issue_template.md:34]: @unknwon added it to patch releases as well.

**Review by unknwon — APPROVED** [review]

File: content/departments/engineering/dev/process/releases/patch_release_issue_template.md [hunk 2]
```diff
@@ -29,6 +29,7 @@ This release is scheduled for **$RELEASE_DATE**.
 
 - [ ] Ensure release configuration in [`dev/release/release-config.jsonc`](https://sourcegraph.com/github.com/sourcegraph/sourcegraph/-/blob/dev/release/release-config.jsonc) on `main` is up-to-date with the parameters for the current release.
 - [ ] Ensure you have the latest version of the release tooling and configuration by checking out and updating `sourcegraph@main`.
+- [ ] Create a `Security release approval issue` and post a message in the [#security](https://sourcegraph.slack.com/archives/C1JH2BEHZ) channel tagging @security-support.
```
Comment (unknwon) [inline content/departments/engineering/dev/process/releases/patch_release_issue_template.md:32]: Actually...

```suggestion
- [ ] Create a [Security release approval](https://github.com/sourcegraph/sourcegraph/issues/new?assignees=andreeleuterio%2C+evict%2C+willdollman%2C+ferozsalam%2C+mohammadualam&labels=release-blocker&template=security-release-approval.md&title=%24RELEASE_VERSION+Security+approval) issue and post a message in the [#security](https://sourcegraph.slack.com/archives/C1JH2BEHZ) channel tagging @security-support.
```

File: content/departments/engineering/dev/process/releases/release_issue_template.md (hunk 1 above)
Comment (unknwon) [inline content/departments/engineering/dev/process/releases/release_issue_template.md:34]:
```suggestion
- [ ] Create a [Security release approval](https://github.com/sourcegraph/sourcegraph/issues/new?assignees=andreeleuterio%2C+evict%2C+willdollman%2C+ferozsalam%2C+mohammadualam&labels=release-blocker&template=security-release-approval.md&title=%24RELEASE_VERSION+Security+approval) issue and post a message in the [#security](https://sourcegraph.slack.com/archives/C1JH2BEHZ) channel tagging @security-support.
```

**Review by unknwon — APPROVED** [review]

**Pull request merged**